clickhouse/service.go

```diff
@@ -931,3 +937,15 @@ func (r *ServiceResource) syncServiceState(ctx context.Context, state *ServiceRe
 
 	return nil
 }
+
+func ServicePasswordUpdateFromPlainPassword(password string) api.ServicePasswordUpdate {
```

Comment:
This was in the `client` file but it's a helper function. Leaving it here for now but it'll likely go away eventually